notebooks/utils/vis.py

```
@@ -0,0 +1,189 @@
+
+
+def plot_effect_estimation(
+    df: pd.DataFrame,
+    x_var: str,
+    x_label: str,
+    y_label: str = "ATE",
+    filename: str = "effect_estimation.png",
+    true_effect_label: str = "True Effect",
+    xticks: list[float] = None,
+    yticks: list[float] = None,
+    colors: list[str] = ["#ec008b", "#e88e2d", "#73bfe2"],
```

Ruff

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)